src/iomusxml.cpp

```diff
@@ -2105,6 +2106,20 @@ void MusicXmlInput::ReadMusicXmlNote(pugi::xml_node node, Measure *measure, std:
                 element = chord;
             }
         }
+        // If the current note is part of a chord.
+        if (nextIsChord || node.select_node("chord")) {
+            if (chord == NULL && m_elementStack.back()->Is(CHORD)) {
+                chord = dynamic_cast<Chord *>(m_elementStack.back());
+            }
```

Comment:
The logic needs to be changed here so the following code (which assumes chord is not NULL) won't execute if the previous element in the stack is not a chord.

Reply:
Since it will never happen (previous element in the stack is always a chord when `node.select_node("chord")`, and `chord` is always set when `nextIsChord`), I just added an `assert(chord);` in d8bbab6e9ba8164ff796cd30bdaf873d131687b5